packages/lit-dev-content/site/docs/v3/ssr/overview.md

```diff
@@ -33,7 +33,7 @@ A number of integrations have also been published which make Lit SSR work out-of
 
 This library is under active development with some notable limitations we hope to resolve:
 
-- Async component work is not supported. See issues [#3219](https://github.com/lit/lit/issues/3219), [#2469](https://github.com/lit/lit/issues/2469).
+- Async component work is not supported. See issue [#2469](https://github.com/lit/lit/issues/2469).
 - Only Lit components using shadow DOM is supported. See issue [#3080](https://github.com/lit/lit/issues/3080).
 - Integration with other SSR frameworks are being worked on. See issues for [NextJS](https://github.com/lit/lit/issues/2391) and [Nuxt](https://github.com/lit/lit/issues/3049).
```

Comment:
While we're making changes, I think we can remove this line.
Instead, can we add to the list of integrations immediately above this section.
```md
- Next.js pages router with [@lit-labs/nextjs](https://www.npmjs.com/package/@lit-labs/nextjs)
- Nuxt 3 with [nuxt-ssr-lit](https://www.npmjs.com/package/nuxt-ssr-lit)
```

Reply:
Done! Good feedback!